apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -5070,7 +5078,27 @@ peers(_Config) ->
         rpc(aec_peers, get_random, [all])),
 
     {ok, 200, #{<<"blocked">> := [], <<"peers">> := []}} = get_peers(),
+    ok.
 
+disabled_debug_endpoints(_Config) ->
+    ?assertMatch(true, rpc(?NODE, aehttp_app, enable_internal_debug_endpoints, [])),
+    {ok, Internal} = rpc(?NODE, application, get_env, [aehttp, internal]),
+    NewInternal = proplists:delete(debug_endpoints, Internal),
+    ok = rpc(?NODE, application, set_env, [aehttp, internal, NewInternal]),
+    ?assertMatch(false, rpc(?NODE, aehttp_app, enable_internal_debug_endpoints, [])),
```

Comment:
I see this test is changing state of node that is carried over to other test case. So tests become order dependent. And I see `enabled_debug_endpoints` depends on true, and I did not find documentation on order of test cases (I did not find in doc how `sequence` be inherited somehow). I propose:
* A. Add option sequence to group `debug_endpoints` - mainly for readability. Or
* B. Move at least the initial `?assertMatch(true, rpc(?NODE, aehttp_app, enable_internal_debug_endpoints, []))` in the init_per_testcase of `disabled_debug_endpoints`, and in end_per_testcase of `disabled_debug_endpoints` restore the config (may require carrying state from the init).

A is quicker, B is cleaner (also for case of test failure).

I suggest A, or at least clarifying in a comment how the order is guaranteed (so to avoid reader to consider this as test intermittently failing).